app/experimenter/nimbus-ui/src/components/FormBranches/index.test.tsx

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { render, screen, cleanup } from "@testing-library/react";
+
+import {
+  SubjectBranch,
+  SubjectBranches,
+  MOCK_EXPERIMENT,
+  MOCK_BRANCH,
+} from "./mocks";
+
+describe("FormBranches", () => {
+  afterEach(() => {
+    cleanup();
```

Comment:
I [think](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup) you should be able to forgo `cleanup`.